seagulls-space-shooter-demo/src/seagulls/space_shooter/_space_rock.py

```diff
@@ -45,7 +44,6 @@ def render(self, surface: Surface) -> None:
         else:
             surface.blit(self._get_cached_rock_large(), self._position)
 
-    @lru_cache()
     def _get_cached_rock_small(self) -> Surface:
```

Comment:
same here :) just `_get_small_sprite()`? with similar changes to methods below?

Reply:
Sure thing - updated